internal/options/env.go

```diff
@@ -49,6 +49,9 @@ type VSOEnvOptions struct {
 
 	// ClientCacheNumLocks is VSO_CLIENT_CACHE_NUM_LOCKS environment variable option
 	ClientCacheNumLocks *int `split_words:"true"`
+
+	// CleanupOrphanedShadowSecretInterval is VSO_HVSA_CLEANUP_ORPHANED_SHADOW_SECRETS_INTERVAL environment variable option
+	HVSACleanupOrphanedShadowSecretInterval time.Duration `split_words:"true"`
```

Comment:
Then a similar thought here to match the command line flag:

```suggestion

	// CleanupShadowSecretIntervalHVSA is VSO_CLEANUP_SHADOW_SECRETS_INTERVAL_HVSA environment variable option
	CleanupShadowSecretIntervalHVSA time.Duration `split_words:"true"`
```